static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-interview.scss

```diff
@@ -8,5 +8,5 @@
 
 @include overide-interview-headline-colours(news, $live-garnett-main-1, $live-garnett-main-2);
 @include overide-interview-headline-colours(sport, $live-garnett-sport1, $live-garnett-sport2);
-@include overide-interview-headline-colours(arts, $live-garnett-arts1, $live-garnett-arts2);
+@include overide-interview-headline-colours(culture, $live-garnett-culture1, $live-garnett-culture2);
```

Comment:
@zef - we need to pass "arts" to the mixin `overide-interview-headline-colours` instead of "culture".